### ISSUE TITLE
AI has unlimeted money

### ISSUE THREAD

**Issue by Optimus1008** [SkaceKamen/random-infantry-skirmish#20]
Well, after my last post I noticed that the AI has unlimeted money.
Everytime I choose a nice little Pawnee (at the beginning of a new round!) I get instantly killed by a Kajman or a Xi'an.
This bug seems to have appeared some days ago (idk a specific date, because I haven't played for a while).

**Comment by SkaceKamen** [comment]
AI buys all vehicles for same price, so they can have the most powerful vehicles at the start if you set starting money high. This makes the enemy vehicles more variable and allows AI to stand a chance against player, which can use his vehicles much more effectively.

When I was adding AI vehicles to the mode, I tested several ways of vehicles spawning.

The normal approach, where AI would have same money and income as player didn't really work, the AI usually only made enough money to buy the most basic vehicles and that wasn't fun at all. Even when I tried to change the multipliers, the AI still usually only used cheap vehicles, since they simply don't get enough kills.

So that's why I decided to change the prices for AI to be same, allowing them to catch up with player.
